apps/frontend/src/views/BaseView.vue

```diff
@@ -50,6 +50,6 @@ const BaseProps = Vue.extend({
 })
 export default class Base extends BaseProps {
   /** Models if the drawer is open */
-  drawer: boolean = false;
+  drawer: boolean = true;
```

Comment:
This changes the default behavior of the app to open the drawer on load. Is this intended?

Reply:
Yes, this is intentional.  I had a chat with @ejaronne and the behavior should be the user will explicitly close the sidebar regardless of switching views.